merge_source_location/P1208.bs

```diff
@@ -141,24 +140,24 @@ private:
 };
 </pre>
 
-Alternatively, source_location could return a const reference:
+Alternatively, `source_location::current()` could return a const reference:
 
 <pre>
 struct source_location {
-    constexpr const source_location & current() noexcept;
+    static constexpr const source_location & current() noexcept;
 };
 </pre>
 
-The authors strongly prefer the first solution as we thing retaining value semantic is important.
+The authors strongly prefer the first solution as we think retaining value semantics is important.
 It is also important to note that, while not implemented, the first solution has been considered and is deemed
 realistically implementable by both GCC and Clang developers (Jonathan Wakely, Eric Fiselier).
 
 ## Removing `current()` altogether ?
 
 If LEWG elects to keep value semantics, the authors would like to propose that the `current` function be removed
-since, in its current form, `source_location` has a default constructor that has no meaningful use.
-It is important to note that `source_location` is not, nor should be, regular as it can not be meaningfully be compared.
-Besides being harder to miss-use, using the constructor of the type to acquire its value is also significantly less verbose.
+since, in its current form, `source_location` has a default constructor with no meaningful use.
+We propose that `source_location` should not be regular, as it cannot meaningfully be compared.
```

Comment:
I strongly disagree with this statement. I believe that `source_location` ought to have an `operator==` that does the obvious thing: two `source_location` values are equal if they represent the same location in the source. I don't understand how you can possibly say that positions cannot "meaningfully be compared."